src/Microsoft.AspNet.Server.Kestrel/Http/Connection.cs

```diff
@@ -65,7 +66,7 @@ public void Start()
                 SocketInput = _rawSocketInput;
                 SocketOutput = _rawSocketOutput;
 
-                _frame = CreateFrame();
+                _frame = CreateFrame(clientCertificate: null);
```

Comment:
Use the same approach as _remoteEndPoint. Store the client cert locally and then pass it through to each frame.